src/django/api/serializers.py

```diff
@@ -226,9 +226,17 @@ def get_is_verified(self, user):
 
 
 class FacilityListSummarySerializer(ModelSerializer):
+    contributor_id = SerializerMethodField()
+
     class Meta:
         model = FacilityList
-        fields = ('id', 'name', 'description')
+        fields = ('id', 'name', 'description', 'contributor_id')
+
+    def get_contributor_id(self, facility_list):
+        try:
+            return facility_list.source.contributor.id
+        except Contributor.DoesNotExist:
+            return False
```

Comment:
Consider returning `None` rather than `False` here. For a function that returns a number, `None` is a more natural "couldn't find it" value.